.changeset/six-pianos-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-router-dom": major
```

Comment:
```suggestion
"react-router-dom": patch
```

This can just be a patch - no new runtime functionality added, just exporting a missing existing type so it's more of a type bugfix IMO